packages/core/tooling/js/vite.ts

```diff
@@ -17,11 +17,17 @@ function exportDefaultConfig(
 		fallbackExpression = object.create({});
 	}
 
-	const { value: rootObject } = exports.createDefault(ast, { fallback: fallbackExpression });
+	const { value: _rootObject } = exports.createDefault(ast, { fallback: fallbackExpression });
 
 	// Handle wrapper functions (e.g., defineConfig({})) if ignoreWrapper is specified
 	let configObject: AstTypes.ObjectExpression;
 
+	// Handle TypeScript satisfies expressions
+	let rootObject = _rootObject;
```

Comment:
Having `rootObject` and `_rootObject` seems unintuitive, Why not change the `const` to a `let` above? or find a better name